Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Use -e ECHO_BOT_TOKEN="Bot exampletoken" as an environment variable when running
+FROM ubuntu:20.04 AS build
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt update && apt install -y \
+    libboost-all-dev \
+    cmake \
+    build-essential \
+    libssl-dev \
+    libcrypto++-dev \
```

Comment:
I don't believe libcrypto++ is needed

Reply:
Correct, seems to be the same thing for libcurl4-openssl